controls/datagrid/aggregates/delegate-aggregate-descriptor.md

```diff
@@ -14,9 +14,23 @@ The `DelegateAggregateDescriptor` allows you to define property lookup logic and
 To set up the `DelegateAggregateDescriptor`, use the following properties:
 
 * `ValueLookup`&mdash;Defines an `IKeyLookup` instance, which retrieves the value from the underlying `ViewModel` that is used for computing the aggregated value.
+
 * `Function`&mdash;Defines an `IAggregateFunction` instance that performs the aggregation of the values as specified by the `ValueLookup` property.
+
 * `Format`&mdash;Defines the string format that will be applied over the aggregated value.
 
+The `IAggregateFunction` interface exposes the following methods:
+
+* `GetValue()`&mdash;Gets the computed value.
+
+* `Accumulate(object value)`&mdash;Applies the function logic to the provided value. Extracted value from the `ViewModel`.
```

Comment:
The second sentence is incomplete and hence unclear. Do you mean that the provided value is extracted from the ViewModel?